wixcode-crm/wix-crm.service.json

```diff
@@ -556,4 +555,4 @@
       { "title": "Why are my custom fields not working when I use 'createContact()'?",
         "markdown": "You can only set values for custom fields that already exist in your site's Contacts List. [Add the custom fields](https://support.wix.com/en/article/adding-custom-fields-to-contacts) in your site's dashboard before using them in code." },
       { "title": "Why are my labels not working when I use 'createContact()'?",
-        "markdown": "You can only set labels that already exist in your site's Contact List. [Create the labels](https://support.wix.com/en/article/creating-contact-labels) in your site's dashboard before using them in code." } ] }
+        "markdown": "You can only set labels that already exist in your site's Contact List. [Create the labels](https://support.wix.com/en/article/creating-contact-labels) in your site's dashboard before using them in code." } ] }
```

Comment:
Did anything change here? Just make sure this was intended.

Reply:
Nevermind I think it's just a line change

Reply:
It was an end of file change, looks like.